src/Microsoft.AspNetCore.SignalR/GetHttpContextExtensions.cs

```diff
@@ -18,6 +26,11 @@ public static HttpContext GetHttpContext(this HubCallerContext connection)
             return connection.Features.Get<IHttpContextFeature>()?.HttpContext;
         }
 
+        /// <summary>
+        /// Gets <see cref="HttpContext"/> from the specified connection.
+        /// </summary>
+        /// <param name="connection">The connection.</param>
+        /// <returns>The connection's <see cref="HttpContext"/>.</returns>
```

Comment:
"The [HttpContext] for the connection, or [null] if the connection is not associated with an HTTP request"